buildpack/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM golang:1.11.0
+LABEL source=https://github.com/kyma-project/test-infra.git
+
+WORKDIR /
+
+## NODE.JS
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get update && apt-get install -y nodejs uidmap libseccomp-dev
+
+RUN nodejs -v
+
+## GOLANG
+# install dep
+RUN go get -u github.com/golang/dep/cmd/dep
+
+# install gometalinter
+RUN go get -u github.com/alecthomas/gometalinter \
+        && gometalinter --install
+
+# install go2xunit
+RUN go get -u github.com/tebeka/go2xunit
+
+# install mockery
+RUN go get -u github.com/vektra/mockery/cmd/mockery
+
+# install linters, prettier, and whitesource 
+RUN npm install -g eslint-config-react-app \
+	babel-eslint@^7.2.3 \
+	eslint@^4.1.1 \
+	eslint-plugin-flowtype@^2.34.1 \
+	eslint-plugin-import@^2.6.0 \
+	eslint-plugin-jsx-a11y@^5.1.1 \
+	eslint-plugin-react@^7.1.0 \
+	tslint \
+	tslint-angular \
+	tslint-config-prettier \
+	typescript \
+	prettier \
+	whitesource  
+
+## Install gcloud
+
+ENV PATH=/google-cloud-sdk/bin:/workspace:${PATH} \
+    CLOUDSDK_CORE_DISABLE_PROMPTS=1
+
+RUN wget -q https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz && \
+    tar xzf google-cloud-sdk.tar.gz -C / && \
+    rm google-cloud-sdk.tar.gz && \
+    /google-cloud-sdk/install.sh \
+        --disable-installation-options \
+        --bash-completion=false \
+        --path-update=false \
+        --usage-reporting=false && \
+    gcloud components install alpha beta kubectl && \
+    gcloud info 
+
+#
+# BEGIN: DOCKER IN DOCKER SETUP
+#
+
+# Install Docker deps
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
if we want to make the image smaller then please run at the end:
```
    && apt-get autoremove -y \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
```
of course in every apt-get install (maybe we can also combine all those apt-get install into one to have less image layers?)

Reply:
Done